instrumentation/aws-sdk/aws-sdk-1.11/testing/src/main/groovy/io/opentelemetry/instrumentation/awssdk/v1_11/AbstractSqsTracingTest.groovy

```diff
@@ -109,16 +105,7 @@ abstract class AbstractSqsTracingTest extends InstrumentationSpecification {
             "net.transport" IP_TCP
           }
         }
-        def offset = 0
-        if (hasHttpClientSpan()) {
-          offset = 1
-          span(1) {
-            name "HTTP POST"
-            kind CLIENT
-            childOf span(0)
-          }
-        }
```

Comment:
The AWS `SQS.SendMessage` span implements both RPC and HTTP conventions, thus it suppresses this HTTP span -- they both had the exact same net/HTTP attributes anyway.

Reply:
@anuraaga this sounds like a good thing, just wanted to get your 👀 on it